openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -1,56 +1,54 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% trans "Assignment in progress" %}
-    </h3>
+    <h4 class="message__title">{% trans "Progress" %}</h4>
```

Comment:
@catong  This is the header for the status that show above the first step.  It is an Overall status.  @cptvitamin Asked that I set it to Progress.  Would "Overall Status" work better?